templates/base.html.twig

```diff
@@ -75,6 +75,22 @@
 					{{ msg( 'issue-button' ) }}
 				</a>
 			</p>
+			<form method="get" class="form-inline change-lang">
+				<p>
+					<select name="uselang" class="form-control">
+						{% for code,lang in all_langs() %}
+							<option value="{{code}}" {% if code == lang() %}selected{% endif %}>
+								{% if lang %}
+									{{lang}}
+								{% else %}
+									({{code}})
```

Comment:
This fallback shouldn't really be necessary, but not all languages have names in Intuition yet. (I think [T331584](https://phabricator.wikimedia.org/T331584) is the task for that, and perhaps `all_langs()` should handle the fallback for us; that's part of the [ToolforgeBundle](https://github.com/wikimedia/ToolforgeBundle).)